app/controllers/api/v1/users_controller.rb

```diff
@@ -16,10 +18,23 @@ def create
     end
   end
 
+  # PATCH/PUT /users/1
+  def update
+    if @user.update(user_params)
+      render json: @user, status: :ok
+    else
+      render json: @user.errors, status: :unprocessable_entity
+    end
+  end
+
   private
 
   # Only allow a trusted parameter "white list" through.
   def user_params
     params.require(:user).permit(:email, :password)
   end
+
+  def set_user
```

Comment:
Please change this method as:
```
def user
  @user ||= User.find(params[:id])
end
```
With this suggestion you don't need to set the user each time that you use the method.